zone/zoning.cpp

```diff
@@ -1224,38 +1224,55 @@ bool Client::CanBeInZone() {
 		return (true);
 	}
 
-	float safe_x, safe_y, safe_z, safe_heading;
-	int16 min_status = AccountStatus::Player;
-	uint8 min_level = 0;
-
 	auto z = GetZoneVersionWithFallback(
-		ZoneID(zone->GetShortName()),
-		zone->GetInstanceVersion()
+		zone_short_name.empty() ? ZoneID(zone->GetShortName()) : ZoneID(zone_short_name),
+		instance_version == -1 ? zone->GetInstanceVersion() : instance_version
 	);
+
 	if (!z) {
 		return false;
 	}
 
-	safe_x       = z->safe_x;
-	safe_y       = z->safe_y;
-	safe_z       = z->safe_z;
-	safe_heading = z->safe_heading;
-	min_status   = z->min_status;
-	min_level    = z->min_level;
+	if (GetLevel() < z->min_level) {
+		LogDebug(
```

Comment:
Let's change all of these `LogDebug`'s to `LogInfo`

`LogDebug` is nice during development but it should probably live somewhere, either a category oriented log sink or just the default "Info"

This information would be useful to at the very minimum see in the info category